davenmathews/Bright_Events
# #153181323 Implements user registration

**Description of work to be completed:**
The following api endpoint is working:
POST /api/v1/register

**How can it be tested?**
After cloning the repo, cd into it and create a virtual environment.
Ensure the users.json file has empty list, [ ] and add it to the virtual environment folder.
Run app.py.
Test in a web browser eg. Chrome and the users created should be seen in the users.json file.
Should add '/home' after the local host on the url to show the UI

**Constraints**
A user will not be created if there exists a user in the users.json file with same email address

**Finishes and Delivers #153181323 story**
